subsys/net/l2/ethernet/gptp/gptp_messages.c

```diff
@@ -14,7 +14,7 @@ LOG_MODULE_DECLARE(net_gptp, CONFIG_NET_GPTP_LOG_LEVEL);
 #include "gptp_md.h"
 #include "gptp_private.h"
 
-#define NET_BUF_TIMEOUT MSEC(100)
+#define NET_BUF_TIMEOUT K_MSEC_TO_TICKS(100)
```

Comment:
I just realized that the code should have used K_MSEC(100) here as the value is suppose to be in milliseconds. I will send a separate patch after this is merged to fix it. So yet another reason to get rid of confusing MSEC().

Reply:
I will change here MSEC (which are actually now K_MSEC_TO_TICKS) to K_MSEC, no problem.

Reply:
Ok, thanks!

But I wonder if we should just remove the SECOND(), MSEC() etc and not create the TICKS version. To me it looks like that all the places where these ticks version are used expect milliseconds and not ticks.